src/Clients/HotChocolate/HotChocolateProvidersDescriptor.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using Thor.Core.Abstractions;
+
+namespace Thor.HotChocolate
```

Comment:
Perhaps we should calling the package/namespace `Thor.Extensions.HotChocolate` so that we and our customers can easily see that this is an extra plugin for a specific case.

Reply:
I can do the renaming in another PR. Also for AspNetCore maybe we should change it to Thor.Hosting.AspNetCore

Reply:
#64 opened